pims/cine.py

```diff
@@ -586,6 +730,97 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def save_image_sequence(self, base_dir=None, im_ext='.tif', fmt='06d',
+            starts_with=1, method='skimage', crop=True, out_range='uint16'):
+        """
+        Burst and save .cine file into image sequence, following selected method.
+
+        Parameters
+        ----------
+        base_dir : None or str, optional.
+            Base directory into which create save subfolder. If None (default), 
+            take the path of the cine file.
+
+        im_ext : str, optional.
+            Single image extension. Default is tif, other values may lead to 
+            unsuspected behaviour (compression, etc).
+
+        fmt : str, optional.
+            String formatting of image labeling.
+
+        starts_with : int, optional.
+            First saved image number, default 1.
+
+        method : str, optional.
+            Choose between 'skimage' (default), 'ffmpeg'.
+            See description of formats in Notes.
+
+        crop : bool, optional.
+            Crop the output image series (requires data to be stored in the .cine).
+
+        out_range : str.
+            Image data type.
+        
+        Notes
+        -----
+        A subfolder containing all frames is systematically created (unless already
+        existing).
+
+        See also
+        --------
+        skimage.exposure.rescale_intensity
+        """
+        # Init file, folder names
+        base, fname = split(self._filename)
+        if base_dir is not None:
+            base = base_dir
+        prefix, ext = splitext(fname)
+        fol = join(base, prefix)
+        try:
+            os.mkdir(fol)
+        except (FileExistsError):
+            ans = input('Folder {} already exists. Overwrite? (y/n) '.format(fol))
```

Comment:
This has to use a different approach because PIMS is often used for batch processing, not interactively. Perhaps there could be a keyword argument that controls this behavior? One of the choices could be `'ask'`, but I'm not sure that it should be the default.